src/chains/base.ts

```diff
@@ -36,3 +37,31 @@ export const base: OpStackChain & { opStackConfig: { l1: { chainId: typeof mainn
     },
   },
 } as const satisfies OpStackChain
+
+export const baseAddresses: Addresses<1> = {
+  optimismPortal: {
+    address: '0x49048044D57e1C92A77f79988d21Fa8fAF74E97e',
+    chainId: 1,
+    blockCreated: 17482143,
```

Comment:
@jxom thoughts on this vs. [ChainContract](https://github.com/wagmi-dev/viem/blob/ab9eec96a33e3dc349783e382f13216debd5f5b8/src/types/chain.ts#L69-L72)? Types are very close? Wonder how you'll resolve when it comes into viem.

Reply:
In general I think this will be fine. These objects can keep being used by folks who need them and migrating to passing in the chain id explicitly is an easy change.